doc/api/process.md

```diff
@@ -630,7 +630,17 @@ process.
 ```js
 console.log(`Current directory: ${process.cwd()}`);
 ```
+## process.debugPort
+<!-- YAML
+added: v0.7.2
+-->
+* {number}
 
+The port used by Node's debugger when enabled.
```

Comment:
It is actually called `Node.js` so it should be called `Node.js's`.

Reply:
Noted